rootdir/vendor/etc/init/spu.rc

```diff
@@ -0,0 +1,8 @@
+service spu_service /odm/bin/hw/vendor.qti.spu@1.0-service
+    class hal
+    user system
+    group system
+    disabled
+
+on boot
+    start spu_service
```

Comment:
Eww, can't they put it in the right class... Or at least `enable`!!! it instead of `start`ing so that it restarts when it _inevitably_ crashes.

Reply:
Please just sure `class core` and remove this + `disabled` BS.

https://sx.ix5.org/info/android-init-stages/

Reply:
It's like that on purpose... this service is essential for security... that's why....

Reply:
If you removed the `disabled`, there's no need to keep the `on boot` trigger since `class_start hal`/`class_start_post_data` will occur before the `on boot` target is reached.

Reply:
@ix5 the stock rc file does it like this, `class hal` with `on boot start`. But apparently `class hal` starts earlier than `boot` making this useless?

Reply:
Stock idiocy is not the benchmark ;)

Reply:
> Stock idiocy is not the benchmark ;)

It isn't our intent to match with stock; rather this file seems to be copied from stock but the `disabled` appeared out of thin air...